ring/replication_set.go

```diff
@@ -316,7 +316,7 @@ func DoUntilQuorumWithoutSuccessfulContextCancellation[T any](ctx context.Contex
 			ext.Error.Set(cfg.Logger.Span, true)
 		}
 
-		contextTracker.cancelAllContexts(cancellation.NewErrorf(cause))
```

Comment:
I can't see why the new version is better.

Reply:
It's generally because the linter thinks the non-formatting version should be used, if there's no string formatting.